platforms/android/demo/src/main/java/com/mapzen/tangram/android/MainActivity.java

```diff
@@ -88,6 +191,42 @@ public void onLowMemory() {
         view.onLowMemory();
     }
 
+    @Override public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        int selectedIndex = 7; // custom
+        String sceneText = autoCompleteView.getText().toString();
+        for (int i = 0; i < scenes.length; i++) {
+            if (sceneText.equals(scenes[i])) {
+                selectedIndex = i;
+                break;
+            }
+        }
+        spinner.setSelection(selectedIndex);
+
+        autoCompleteView.clearFocus();
+        InputMethodManager in = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
+        in.hideSoftInputFromWindow(autoCompleteView.getWindowToken(), 0);
+
+        map.loadSceneFile(autoCompleteView.getText().toString(),sceneUpdates);
```

Comment:
Agreed. I would suggest implementing `loadSceneFileAsync` call in our `MapController` to address this issue. But outside the scope of this PR and can be addressed in a separate PR.